monitoring/prometheus/prometheus.yml

```
@@ -0,0 +1,51 @@
+global:
+  scrape_interval: 15s
+  evaluation_interval: 15s
+
+rule_files:
+  - "rules/*.yml"
+
+alerting:
+  alertmanagers:
+    - static_configs:
+        - targets:
+          - alertmanager:9093
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 12 but found 10

(indentation)